pkg/ui/.gitignore

```diff
@@ -8,10 +8,14 @@ yarn.opt.installed
 yarn.cluster-ui.installed
 yarn.protobuf.installed
 yarn-error.log
+assets.ccl.installed
+assets.oss.installed
 
 # Generated intermediates
 .cache-loader
 dist*/**
+!dist*/.gitkeep
```

Comment:
What purpose does the `.gitkeep` file serve? From what I can tell, since we also have the `index.html` file in the same directory, it doesn't need to be there.

Reply:
Correct, it has to be removed, thanks!